scripts/generatePullRequestForNewInitiative.js

```diff
@@ -164,9 +166,10 @@ The value of property \`${prop}\` (\`${value}\`) is already present in \`${linkJ
 \`\`\`json
 ${linksJsonString}
 \`\`\`
-**If this is a mistake and it doesn't already exist:** edit the issue's title so that it starts with **\`[NEW-INITIATIVE-FORCE-PR]:\`**`,
+**If this is a mistake and it doesn't already exist:** edit the issue's title so that it starts with **\`[FORCE-PR-NEW-INITIATIVE]:\`**
+**If you wish to update an existing initiative:** edit the issue's title so that it starts with **\`[UPDATE-INITIATIVE]:\`**`,
```

Comment:
```suggestion
- **If this is a mistake and it doesn't already exist:** edit the issue's title so that it starts with **\`[FORCE-PR-NEW-INITIATIVE]:\`**
- **If you wish to update an existing initiative:** edit the issue's title so that it starts with **\`[UPDATE-INITIATIVE]:\`**`,
```